packages/feetch/package.json

```diff
@@ -34,12 +34,8 @@
     "@testing-library/react-hooks": "3.2.1",
     "@types/jest": "25.1.4",
     "@types/react": "16.9.23",
-    "@typescript-eslint/eslint-plugin": "2.23.0",
-    "@typescript-eslint/parser": "2.23.0",
     "eslint": "6.8.0",
-    "eslint-config-kodefox": "0.1.0",
-    "eslint-config-prettier": "6.10.0",
-    "eslint-plugin-prettier": "3.1.2",
+    "eslint-config-kodefox": "^0.2.2",
     "eslint-plugin-react": "7.19.0",
```

Comment:
I think we can also remove this one

Reply:
Does it because `eslint-config-kodefox` has it as `optionalDep`?

Reply:
Correct!